packages/trader/src/AppV2/Components/TradeParameters/Duration/day.tsx

```diff
@@ -40,6 +40,9 @@ const DayInput = ({
     const [current_gmt_time, setCurrentGmtTime] = React.useState<string>('');
     const [open, setOpen] = React.useState(false);
     const [open_timepicker, setOpenTimePicker] = React.useState(false);
+    const [trigger_date, setTriggerDate] = useState(false);
+    const [is_disabled, setIsDisabled] = useState(false);
+    const [temp_end_date, setTempEndDate] = useState(end_date);
```

Comment:
temp_end_date? what does that name mean? 
could it be just "end_date", or is the reason for it to be "temp_"? (feels bad to put "temp" things to state, but there might be reasons for it which I'm not seeing at the moment)

Reply:
I had the exact same question

Reply:
its good to have naming which immediately tells the reader what does it mean. E.g. in this case - how its different from end_date. Examples: selected_end_date, formatted_end_date, localised_end_date

Reply:
It is changed to `end_date_input` , thanks